src/altinn-app-frontend/src/features/form/components/MessageBanner.test.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { AltinnAppTheme } from 'altinn-shared/theme';
+
+import MessageBanner from './MessageBanner';
+import type { ILanguage } from 'altinn-shared/types';
+
+describe('features > form > components > MessageBanner.tsx', () => {
```

Comment:
nitpick, but I would suggest to keep these simple and just name it `MessageBanner` - the location doesnt really matter when these are logged, the filepath  to the failing test will be output anyway, and the path may change if we move code around.